New branch - 7.x 1.x

dakala opened the pull request:
@simultech: Hope this works. It's my 7.x-1.x branch which is already pushed to drupal.org. If not could we have a 7.x-1.x branch on your repo please? Maybe at the same time we can have an 8.x-1.x branch too. Thanks.

simultech commented:
Works wonderfully, thanks!  Merged.  Apologies, starting to get back into things now, we've been crunching on the new version (getting very close now).

dakala commented:
That's ok. We've been busy at our end too. I'll be going back to our
Shorthand work soon. When I've cleaned up and refactored the code more
maybe we'll move on to a first beta version. I don't expect more than 2
beta versions as what we've currently got works well.

Let me know if I can assist with anything in the new API :-)


Deji




On 29 June 2017 at 07:54, Andrew Dekker <notifications@github.com> wrote:

> Works wonderfully, thanks! Merged. Apologies, starting to get back into
> things now, we've been crunching on the new version (getting very close
> now).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Shorthand/shorthand_connect_drupal/pull/17#issuecomment-311878918>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARbZ2HJ701C5bdtKCNOl7PDr_X5RpPmks5sI0o7gaJpZM4NmHfl>
> .
>
